keps/sig-release/3027-slsa-compliance/README.md

```diff
@@ -0,0 +1,894 @@
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+This document proposes a plan to harden the Kubernetes releases by
+making the necessary adjustments to comply with the SLSA Framework.
+[SLSA (Supply-chain Levels for Software Artifacts)](https://slsa.dev/)
+is a framework to harden software supply currently being defined by the 
+[OpenSSF](https://openssf.org/)'s 
+[Supply Chain Integrity WG](https://github.com/ossf/wg-supply-chain-integrity).
+
+The framework provides requirements and recommendations to software
+build systems to harden their environments and the processes that drive
+them. It also defines the metadata that needs to be produced to trace the 
+origins of every item in a software release.
+
+The main goal of this enhancement is to provide downstream consumers of our
+artifacts the highest assurance about the integrity of each Kubernetes release.
+
+SLSA defines several levels of hardening, each touching more aspects of the
+release process that go beyond its technical implementation. This document is
+meant to serve as a guide to reach the highest possible levels after 
+consensus has been reached about their viability.
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+## Motivation
+
+Release Kubernetes in a zero-trust environment.
```

Comment:
I would make this either a headline or rephrase for a higher verbosity, like:

> The main motivation for reaching full SLSA compliance is to be able to release Kubernetes in a zero-trust environment.

Reply:
OK, changed it :)